dataprep/eda/diff/compute/multiple_df.py

```diff
@@ -0,0 +1,368 @@
+        else:
+            col_dtype = col_dtype[0]
+
+        orig = [src for src, seq in labeled_cols.items() if col in seq]
+
+        if is_dtype(col_dtype, Continuous()) and cfg.hist.enable:
+            data.append((col, Continuous(), _cont_calcs(srs.apply("dropna"), cfg), orig))
+        elif is_dtype(col_dtype, Nominal()) and cfg.bar.enable:
+            srs = srs.apply("astype(str)")
+            data.append((col, Nominal(), _nom_calcs(srs.apply("dropna"), cfg), orig))
+        elif is_dtype(col_dtype, DateTime()) and cfg.line.enable:
+            data.append(
+                (col, DateTime(), dask.delayed(_calc_line_dt)(srs, col, cfg.line.unit), orig)
+            )
+
+    stats = calc_stats(dfs, cfg, dtype)
+    data = [dask.compute(*i) for i in data]
+    stats = {k: list(dask.compute(*v)) for k, v in stats.items()}
+    plot_data: List[Tuple[str, DTypeDef, Any, List[str]]] = []
+
+    for col, dtp, datum, orig in data:
+        if is_dtype(dtp, Continuous()):
+            if cfg.hist.enable:
+                plot_data.append((col, dtp, datum["hist"], orig))
+        elif is_dtype(dtp, Nominal()):
+            if cfg.bar.enable:
+                plot_data.append(
+                    (col, dtp, (dask.compute(*datum["bar"]), dask.compute(datum["nuniq"])[0]), orig)
+                )
+        elif is_dtype(dtp, DateTime()):
+            plot_data.append((col, dtp, (dask.compute(*datum[0]), datum[1]), orig))  # workaround
+    return Intermediate(data=plot_data, stats=stats, visual_type="comparison_grid")
+
+
+def calc_stats(dfs: Dfs, cfg: Config, dtype: Optional[DTypeDef]) -> Dict[str, List[str]]:
+    """
+    Calculate the statistics for plot_diff([df1, df2, ..., dfn])
+
+    Params
+    ------
+    dfs
+        DataFrames to be compared
+    """
+    stats: Dict[str, List[Any]] = {"nrows": dfs.shape.getidx(0)}
+    dtype_cnts = []
+    num_cols = []
+    if cfg.stats.enable:
+        for df in dfs:
+            temp = get_dtype_cnts_and_num_cols(df, dtype=dtype)
+            dtype_cnts.append(temp[0])
+            num_cols.append(temp[1])
+
+        stats["ncols"] = dfs.shape.getidx(1)
+        stats["npresent_cells"] = dfs.apply("count").apply("sum")
+        stats["nrows_wo_dups"] = dfs.apply("drop_duplicates").shape.getidx(0)
+        stats["mem_use"] = dfs.apply("memory_usage(deep=True)").apply("sum")
+        stats["dtype_cnts"] = dtype_cnts
+
+    return stats
+
+
+def _cont_calcs(srs: Srs, cfg: Config) -> Dict[str, List[Any]]:
+    """
+    Computations for a continuous column in plot_diff([df1, df2, ..., dfn])
+    """
+
+    data: Dict[str, List[Any]] = {}
+
+    # drop infinite values
+    mask = srs.apply("isin({np.inf, -np.inf})")
+    srs = Srs(srs.getmask(mask, inverse=True), agg=True)
+
+    # histogram
+    data["hist"] = srs.self_map(
+        da.histogram,
+        bins=cfg.hist.bins,
+        range=(min(dask.compute(*srs.apply("min"))), max(dask.compute(*srs.apply("max")))),
```

Comment:
`range` can take delayed objects as parameters, so I think it would be better if we don't call `compute`.

Reply:
We apply the `min()` and `max()` to multiple columns at the same time here so we will get the minimum value and maximum value of each column, then we need to pick the minimum and the maximum from the list we got where has multiple `dd.Scalar` objects. However, the list here is a `UserList` we defined in advance, so we can't have the list return a minimum or maximum value of many `dd.scalar`s.